defaults/main.yml

```diff
@@ -751,42 +751,46 @@ postgresql_pgdg_releases:
       9.3: 3,
       9.4: 3,
       9.5: 3,
-      9.6: 3,
-       10: 2,
+      9.6: 10,
```

Comment:
These aren't the patch versions, but are the YUM repo versions.  They rarely change once the product has been released GA.